dev-util/cutter/cutter-1.10.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Looking at deps I think you can add 3.8 and 3.9 here.

Reply:
I tried and cutter does not build with python3.8 unfortunately :(
It might be some make system issue as there are only linker errors about not finding python symbols

Reply:
What is even stranger, it seems that it's the fault of `dev-lang/python:3.8`, as (at least on my machine) running `python3.8-config --libs` is not working correctly, it only returns `-lcrypt -lpthread -ldl  -lutil -lm -lm`.

Reply:
It seems to be caused by https://docs.python.org/3.8/whatsnew/3.8.html#debug-build-uses-the-same-abi-as-release-build
I'll try to fix it

Reply:
fixed